docs/samples/advanced_config/aggregator.yaml

```diff
@@ -211,3 +211,13 @@ task_cache_ttl_s: 600
 # Defines how many tasks can be cached. This affects how much memory the
 # aggregator might use to store cached tasks. (optional)
 task_cache_capacity: 10000
+
+# Queue aggregate init and continue requests with a LIFO strategy. (optional;
+# experimental, may be removed or changed in minor releases)
+helper_request_queue:
+  # The number of requests that can be processed concurrently.
+  concurrency: 1
+  # The maximum number of requests that can be left waiting in the queue for
+  # a concurrency slot to become available. Excess requests are rejected with
+  # HTTP status 503 Unavailable. (optional)
```

Comment:
Is this optional? I don't see where the default is set.

IMO, either document the default value if there is a default, or drop the `(optional)`.